pallets/collator-allowlist/src/weights.rs

```diff
@@ -0,0 +1,63 @@
+//! Autogenerated weights for pallet_fees
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 3.0.0
+//! DATE: 2021-06-18, STEPS: `[50, ]`, REPEAT: 100, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("altair-dev"), DB CACHE: 128
+
+// Executed Command:
+// target/release/centrifuge-chain
+// benchmark
+// --chain=altair-dev
+// --steps=50
+// --repeat=100
+// --pallet=pallet_fees
+// --extrinsic=*
+// --execution=wasm
+// --wasm-execution=compiled
+// --heap-pages=4096
+// --output=./pallets/collator-allowlist/src/weights.rs
+// --template=./scripts/frame-weight-template.hbs
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{
+	traits::Get,
+	weights::{constants::RocksDbWeight, Weight},
+};
+use sp_std::marker::PhantomData;
+
+/// Weight functions needed for pallet_collator_allowlist.
+pub trait WeightInfo {
+	fn allow() -> Weight;
+	fn remove() -> Weight;
+}
+
+/// The weight associated with allowlist a collator
+const ALLOWLIST_WEIGHT: Weight = 42 as Weight;
+
+/// The weight associated with removing a collator
+const REMOVE_WEIGHT: Weight = 24 as Weight;
```

Comment:
Is this auto-generated? Looks like they updated benchmarking then.

Reply:
No, it's manual. Sorry, I didn't realise that this was meant to be auto-generated, I just copied the one from the fees pallets to get started. I was planning on working on the benchmarking side of things and make these numbers realistic in a follow up PR anyway.

Reply:
To be clear, I used `42` and `24` as random placeholder values here until the right benchmarking weight outputs values were calculated.